### DISCUSSION TITLE
IOS is showing black screen

### DISCUSSION THREAD

**Discussion by jyothine** [software-mansion/react-native-svg#2447] [Q&A]
<img width="377" alt="Screenshot 2024-09-12 at 12 38 14 PM" src="https://github.com/user-attachments/assets/df1005b7-755c-485a-960f-5f69ccb9bfa8">

Getting black screen when we don't add opacity to props

<Svg
    height="50%"
    width="50%"
    viewBox="0 0 100 100"
  >
    <Circle cx="50" cy="50" r="45" stroke="blue" strokeWidth="1" fill="green" />
  </Svg>

If we add opacity we see white screen but no circle is displayed

<Svg
    height="50%"
    width="50%"
    viewBox="0 0 100 100"
     opacity={0}
  >
    <Circle cx="50" cy="50" r="45" stroke="blue" strokeWidth="1" fill="green" />
  </Svg>

 react-native - "0.74.1"
react-native-svg -  "15.3.0",

**Comment by bohdanprog** [comment]
Hello @jyothine , can you create a repo with a simple reproduction of that problem?
Because I can't reproduce that problem.

**Comment by jakex7** [comment]
Closing this as it's a duplicate of #2448 and Issues are a better place to track this type of problem. Please feel free to follow the discussion there.
